src/components/Cluster/ClusterDetail/ClusterApps.js

```diff
@@ -309,22 +312,63 @@ class ClusterApps extends React.Component {
           </p>
           <div className='row'>
             {this.props.release ? (
-              Object.keys(this.preinstalledApps()).map(appCategory => {
-                return (
-                  <div className='col-4' key={appCategory}>
-                    <h6>{appCategory}</h6>
-                    {this.preinstalledApps()[appCategory].map(app => {
-                      return (
-                        <div className='cluster-apps--app' key={app.name}>
-                          <img alt={`${app.title} icon`} src={app.logoUrl} />
-                          {app.name}
-                          <small>{app.version}&nbsp;</small>
-                        </div>
-                      );
-                    })}
-                  </div>
-                );
-              })
+              <>
+                <div className='col-4' key='essentials'>
+                  <h6>essentials</h6>
+                  {this.preinstalledApps().essentials.map(app => {
+                    return (
+                      <div className='cluster-apps--app' key={app.name}>
+                        <img alt={`${app.title} icon`} src={app.logoUrl} />
+                        {app.name}
+                        <small>{app.version}&nbsp;</small>
+                      </div>
+                    );
+                  })}
+                </div>
+
+                <div className='col-4' key='management'>
+                  <h6>management</h6>
+                  {this.preinstalledApps().management.map(app => {
+                    return (
+                      <div className='cluster-apps--app' key={app.name}>
+                        <img alt={`${app.title} icon`} src={app.logoUrl} />
+                        {app.name}
+                        <small>{app.version}&nbsp;</small>
+                      </div>
+                    );
+                  })}
+                </div>
```

Comment:
These 2 are really similar. Do you mind creating a separate component for them?

Reply:
The ingress column is similar, as well

Reply:
ok! will fix that